app/Providers/AuthServiceProvider.php

```diff
@@ -76,5 +78,15 @@ class AuthServiceProvider extends ServiceProvider
     public function boot(): void
     {
         $this->registerPolicies();
+
+        if (! $this->app->routesAreCached()) {
+            Passport::routes();
+        }
+
+        Passport::useClientModel(PassportClient::class);
+        Passport::hashClientSecrets();
+        Passport::tokensExpireIn(now()->addDays(15));
+        Passport::refreshTokensExpireIn(now()->addDays(30));
+        Passport::personalAccessTokensExpireIn(now()->addMonths(6));
```

Comment:
I believe you said in Slack that you'd either extend this or add a job warning the user when tokens are about to expire.